--- 11.Ruth.orig	2017-03-09 10:01:20.000000000 -0500
+++ 11.Ruth.mlxx	2017-03-09 10:17:25.000000000 -0500
@@ -462,7 +462,7 @@
 POREU/ESQAI              V1  PMN    POREU/OMAI
 MET'                     P          META/
 AU)TH=S                  RD  GSF    AU)TO/S
-E)KO/PASEN               VAI AAI3S  KOPAZ/W
+E)KO/PASEN               VAI AAI3S  KOPA/ZW
 TOU=                     RA  GSN    O(
 LALH=SAI                 VA  AAN    LALE/W
 PRO\S                    P          PRO/S
@@ -2239,4 +2239,3 @@
 E)GE/NNHSEN              VAI AAI3S  GENNA/W
 TO\N                     RA  ASM    O(
 *DAUID                   N   ASM    *DAUI/D
-
