--- 42.Jonah.orig	2016-12-11 16:24:17.000000000 -0500
+++ 42.Jonah.mlxx	2016-12-11 16:24:40.000000000 -0500
@@ -628,7 +628,7 @@
 
 Jonah 2:11
 KAI\                     C          KAI/
-PROSETA/GH               VZI AAI3S  TA/SSW           PROS
+PROSETA/GH               VZI API3S  TA/SSW           PROS
 TW=|                     RA  DSN    O(
 KH/TEI                   N3E DSN    KH=TOS
 KAI\                     C          KAI/
@@ -1183,4 +1183,3 @@
 KAI\                     C          KAI/
 KTH/NH                   N3E NPN    KTH=NOS
 POLLA/                   A1  NPN    POLU/S
-
